### PR TITLE
Feature/add event

### PR DESCRIPTION
**Pull request by NicolasMatekalo**
A user can now add an event.
If he owns an event, he can modify it.

### CODE REVIEW

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/fragment/navigation/EventFragmentTest.kt [hunk 1]
```diff
@@ -71,4 +82,33 @@ class EventFragmentTest {
...
+    fun clickOnAddEventButtonLoggedOutDoesNotWork() {
+        FakeConnectedUser.instance = FakeConnectedUser.Instance.LOGGED_OUT
+        val intent = Intent(ApplicationProvider.getApplicationContext(), MainActivity::class.java)
+        val scenario: ActivityScenario<MainActivity> = ActivityScenario.launch(intent)
+        onView(withId(R.id.activityMainBottomNavigationView)).perform(
+            navigateTo(R.id.event)
+        )
+
+        onView(withId(R.id.addEventButton)).perform(click())
+        Thread.sleep(2000)
```
Comment (mbouilloud) [inline app/src/androidTest/java/com/github/sdp/ratemyepfl/fragment/navigation/EventFragmentTest.kt:109]: Is this sleep useful for the test ?

**Review by jmettler27 — APPROVED** [review]
Nothing to say, clean code, great feature.

File: app/src/androidTest/java/com/github/sdp/ratemyepfl/fragment/navigation/EventFragmentTest.kt (hunk 1 above)
Comment (NicolasMatekalo) [reply app/src/androidTest/java/com/github/sdp/ratemyepfl/fragment/navigation/EventFragmentTest.kt:109]: Yes! It waits for the snackbar to disapear so the check on the add button is sure to pass.

File: app/src/main/java/com/github/sdp/ratemyepfl/layout/CustomScrollView.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    constructor(context: Context?) : super(context)
+    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
+    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
+        context,
+        attrs,
+        defStyle
+    )
+
+    fun addInterceptScrollView(view: View) {
+        Log.i(null, "issou add")
```
Comment (mbouilloud) [inline app/src/main/java/com/github/sdp/ratemyepfl/layout/CustomScrollView.kt:26]: You forgot this line :)

**Review by mbouilloud — APPROVED** [review]
Clean code, really awesome UI. Good job.

File: app/src/main/java/com/github/sdp/ratemyepfl/layout/CustomScrollView.kt (hunk 2 above)
Comment (NicolasMatekalo) [reply app/src/main/java/com/github/sdp/ratemyepfl/layout/CustomScrollView.kt:26]: Yes indeed...

**Pull request merged**